health-integrations/src/main/java/com/netflix/runtime/health/servlet/HealthStatusServlet.java

```diff
@@ -26,23 +26,31 @@
 
 import com.netflix.runtime.health.api.HealthCheckAggregator;
 import com.netflix.runtime.health.api.HealthCheckStatus;
+import com.netflix.runtime.health.api.IndicatorFilter;
+import com.netflix.runtime.health.status.ArchaiusHealthStatusFilterModule;
```

Comment:
Apparently that is from my linking the module in some of the javadoc. Thanks for pointing that out.